buildlib/sanitize_static_lib.py

```diff
@@ -144,7 +144,9 @@ def __init__(self, libfn, tmpdir):
         os.makedirs(self.final_objdir)
 
         subprocess.check_call([args.ar, "x", libfn], cwd=self.objdir)
-        self.objects = [I for I in os.listdir(self.objdir)]
```

Comment:
This is just sorted(os.listdir(self.objdir))

But why do we want to do this? Please explain in the commit message

Also use force push to update your branches not close/reopen.